src/webgpu/web_platform/copyToTexture/video.spec.ts

```diff
@@ -41,16 +41,18 @@ It creates HTMLVideoElement with videos under Resource folder.
   .params(u =>
     u //
       .combineWithParams(kVideoExpectations)
+      .combine('useVideoFrame', [true, false])
       .combine('srcDoFlipYDuringCopy', [true, false])
   )
   .fn(async t => {
-    const { videoName, srcDoFlipYDuringCopy } = t.params;
+    const { videoName, useVideoFrame, srcDoFlipYDuringCopy } = t.params;
 
     const videoElement = getVideoElement(t, videoName);
 
     await startPlayingAndWaitForVideo(videoElement, () => {
-      const width = videoElement.videoWidth;
-      const height = videoElement.videoHeight;
+      const videoFrame = new VideoFrame(videoElement);
```

Comment:
nit: Maybe we could only create the videoFrame if useVideoFrame is true? Also can we gate the usage of it on the constructor being present? I think other tests might do that.

Reply:
Good suggestion. I've addressed it.